spec/system/task_spec.rb

```diff
@@ -1,24 +1,24 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) {create(:project)}
+  let(:task) {create(:task)}
```

Comment:
細かいですがスペースを空けた方が見やすいですね。
```suggestion
  let(:project) { create(:project) }
  let(:task) { create(:task) }
```

Reply:
e3749745ab3315fb5ff2b3dceb7230073dcab735
スペースを追加しました。